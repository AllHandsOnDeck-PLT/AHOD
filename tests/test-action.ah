int x
when do PLAY():
  {
  	do PRINT("no params from this action")
  }
main:
  {
  do PLAY()
  }

