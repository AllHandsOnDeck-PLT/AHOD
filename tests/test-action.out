no params from this action